plugins/system/accessibility/accessibility.php

```diff
@@ -64,6 +82,12 @@ public function onBeforeCompileHead()
         // Detect the current active language
         $lang = Factory::getLanguage()->getTag();
 
+        // Get the item id of the menu item for feedback redirection
+        $menuItemId = $this->getAccessibilityItemId();
+
+        //generate the url to pass it to the accessibility script
+        $url = Route::link("site", "index.php?Itemid={$menuItemId}");
```

Comment:
Use single quote .. compare similar code in joomla

Reply:
converted to single quote